### DISCUSSION TITLE
Are public instances possible?

### DISCUSSION THREAD

**Discussion by userunknown358** [Rehike/Rehike#606] [Q&A] [answered]
### Describe the feature

Can Rehike be hosted as a public instance like how YT2009 supports hosting as a public instance?

**Comment by TorutheRedFox** [comment, marked as answer]
kinda but not really

plus it's heavily advised against since you're sending your youtube token to a 3rd party who can capture it and use it to access your account without your knowledge

**Reply by kawapure** [reply]
Additionally: Rehike does not take considerations for security in multi-user environments. For instance, only one global configuration is supported, which could be reconfigured easily from the GUI by any user of the instance, and planned features like the extension loader would effectively allow any arbitrary code execution in a massive multi-user environment.

Rehike is designed solely as a single-user server.
